iothub/device/tests/RetryPolicies/ExponentialBackoffTests.cs

```diff
@@ -10,26 +11,29 @@ namespace Microsoft.Azure.Devices.Client.Test
     [TestClass]
     public class ExponentialBackoffTests
     {
-        private const int MAX_RETRY_ATTEMPTS = 5000;
+        private const int MaxRetryAttempts = 50;
```

Comment:
I printed out the values and it topped out after 3. With my change it takes no more than a handful still, but 50 is plenty of iterations.